### ISSUE TITLE
Is this result normal?

### ISSUE THREAD

**Issue by senmaoy** [junyanz/pytorch-CycleGAN-and-pix2pix#424]
When I trained the label2image model on cityscape dataset, I got the following results:
Mean pixel accuracy: 0.536839
Mean class accuracy: 0.136477
Mean class IoU: 0.088959
It seems much worse than the original Mean pixel accuracy: 0.63 result.  But when I look through the generated images, they look good.

**Comment by junyanz** [comment]
It looks normal, slightly worse than the numbers in the paper (based on Torch implementation). 
When you train the cityscape label-> image, sometimes the tree and building labels will be flipped, which hurts the numbers thought images look great.
